compiler/optimizer/Inliner.cpp

```diff
@@ -2382,6 +2382,63 @@ TR_ParameterToArgumentMapper::lookForModifiedParameters(TR::Node * node)
       }
    }
 
+void
+TR_ParameterToArgumentMapper::mapOSRCallSiteRematTable(uint32_t siteIndex)
```

Comment:
Can we please have a descriptive comment on what this does and emphasize that it only does something if OSR is enabled? Should there be a check for OSR enablement in here explicitly in addition to testing the remat size?